plugins/woocommerce/client/blocks/assets/js/blocks/featured-items/style.scss

```diff
@@ -153,6 +153,14 @@
 		}
 	}
 
+	&__inner-blocks {
+		a,
```

Comment:
Not an issue now, but if in the future we allow the paragraph block or something similar inside the _Featured Product_ or _Featured Category_ blocks, this selector will affect any links inside the paragraph too.

I guess we could be more specific with something along these lines:

```CSS
.wp-block-post-title,
.wp-block-post-title > a,
.wp-block-woocommerce-category-title,
.wp-block-woocommerce-category-title > a { ... }
```